cypress-tests/cypress/e2e/PayoutUtils/Nomupay.js

```diff
@@ -0,0 +1,195 @@
+const billing = {
+    address: {
+        line1: "1467",
+        line2: "Harrison Street",
+        line3: "Harrison Street",
+        city: "Munich",
+        state: "CA",
+        zip: "80331",
+        country: "DE",
+        first_name: "John",
+        last_name: "Doe"
+    },
+    phone: {
+        number: "9123456789",
+        country_code: "+91"
+    },
+    email: `payout_customer${Date.now()}@example.com`
+};
+
+const bank = {
+    iban: "DE57331060435647542639",
+    bic: "DEUTDE5M551",
+    bank_name: "Deutsche Bank",
+    bank_country_code: "DE",
+    bank_city: "Munich"
+};
+
+const error = {
+  type: "invalid_request",
+  message: `Payout Eligibility for Nomupay is not implemented`,
+  code: "IR_00",
+}
+
+export const connectorDetails = {
+    card_pm: {
+      Create: {
+        Request: {
+          payout_method_data: {
+            card: card_data,
+          },
+          payout_type: "card",
+        },
+        Response: {
+          status: 501,
+          body: {
+            error,
```

Comment:
this should be `error: error` just like `card: card_data` in `L39`. Else the assertion will fail miserably!